Makefile

```diff
@@ -1,9 +1,10 @@
-mocha_option := --recursive -t 5000 -s 100
+travis_option := --recursive -t 5000 -s 100
+local_option := $(travis_option) --compilers js:babel-core/register $(mocha_opts)
```

Comment:
looks like we dont. I still dont know why we need this locally, tests are written in es5.


![image](https://user-images.githubusercontent.com/3425521/28506042-fcab7ccc-705a-11e7-9f91-565119d2ef1d.png)